rumqttc/README.md

```diff
@@ -77,17 +77,25 @@ Quick overview of features
 
 In short, everything necessary to maintain a robust connection
 
-**NOTE**: Looping on `connection.iter()/eventloop.poll()` is necessary to
-run the eventloop and make progress. It yields incoming and outgoing activity
-notifications which allows customization as user sees fit.
-
-**IMPORTANT** Blocking inside `eventloop.poll()/eonnection.iter()` loop
-will block connection progress
-
 Since the eventloop is externally polled (with `iter()/poll()` in a loop)
 out side the library and `Eventloop` is accessible, users can
 - Distribute incoming messages based on topics
 - Stop it when required
 - Access internal state for use cases like graceful shutdown or to modify options before reconnection
 
+## Important notes
+
+- Looping on `connection.iter()/eventloop.poll()` is necessary to run the
+  event loop and make progress. It yields incoming and outgoing activity
+  notifications which allows customization as you see fit.
+
+- Blocking inside the `eventloop.poll()/eonnection.iter()` loop will block
+  connection progress
+
+- You cannot create a TLS connection to a bare IP address with a self-signed
+  certificate. This is a [limitation of Rustls](https://github.com/ctz/rustls/issues/184).
+  One workaround, which only works under *nix/BSD-like systems, is to add an
+  entry to wherever your DNS resolver looks (eg. `/etc/hosts`) for the bare IP
```

Comment:
I feel like "e.g." is more correct than "eg." - being an abbreviation for "exempli gratia" -, but I looked it up and there doesn't seem to be a consensus on this, so it's just my opinion, really - feel free to ignore.

```suggestion
  entry to wherever your DNS resolver looks (e.g. `/etc/hosts`) for the bare IP
```

Reply:
The rule is basically a matter of style nowadays, rather than correctness. Happy to change it.